misc/tracing-extensions/modules/ballerina-jaeger-extension/src/main/java/org/ballerinalang/observe/trace/extension/jaeger/OpenTracingExtension.java

```diff
@@ -48,11 +49,25 @@
 @JavaSPIService("org.ballerinalang.util.tracer.OpenTracer")
 public class OpenTracingExtension implements OpenTracer {
 
+    private static final PrintStream console = System.out;
     private static final PrintStream consoleError = System.err;
+    private Map<String, String> configProperties;
 
     @Override
-    public Tracer getTracer(String tracerName, Map<String, String> configProperties, String serviceName)
-            throws InvalidConfigurationException {
+    public void init(Map<String, String> configProperties) {
+        console.println("ballerina: started publishing tracers to Jaeger on "
```

Comment:
How do you confirm whether Ballerina successfully publishes tracers to Jaeger?

Reply:
Jaeger does not throw any exception. Rather drops the span if it can not connect to the Jaeger server